internal/server/stat/golden/get_stat_all/result.json

```diff
@@ -1333,18 +1341,6 @@
           "sourceSeries": [
             {
               "val": {
-                "2008": 2116,
-                "2009": 2155,
-                "2010": 1633,
-                "2011": 1509,
-                "2012": 1581,
-                "2013": 1867,
-                "2014": 1770,
-                "2015": 2201,
-                "2016": 1913,
-                "2017": 2138,
-                "2018": 2267,
-                "2019": 2634,
```

Comment:
@manishvats2 @Bipnabraham 
Is FBIGovCrime expected to have deleted old data from 2008-2019?
Can we have this retained from earlier import as historical data?